mycroft/configuration/ovos.py

```diff
@@ -26,6 +26,10 @@
    //      eg, ovos.conf, chatterbox.conf, neon.conf...
    // "mycroft.conf" paths are derived from this
    //        ~/.{base_folder}/{config_filename}
+   // supported file extensions:
+   //  - json (.json)
+   //  - commented json (.conf)
+   //  - yaml (.yml) are supported
```

Comment:
this should probably be `.yaml` per the actual spec https://yaml.org/faq.html

I don't think it matters much though since the extensions are arbitrary

Reply:
both are supported here, but should update this docstr